code/modules/mob/living/living.dm

```diff
@@ -1119,7 +1119,7 @@
 		for(var/path in butcher_results)
 			for(var/i = 1 to butcher_results[path])
 				new path(src.loc)
-			butcher_results.Remove(path) //In case you want to have things like simple_animals drop their butcher results on gib, so it won't double up below.
+			butcher_results.Remove(path)	//In case you want to have things like simple_animals drop their butcher results on gib, so it won't double up below.
```

Comment:
```suggestion
			//In case you want to have things like simple_animals drop their butcher results on gib, so it won't double up below.
			butcher_results.Remove(path)
```